quaterion_models/utils/meta.py

```diff
@@ -16,6 +16,6 @@ def merge_meta(meta: Dict[str, list]) -> List[dict]:
         if aggregated is None:
             aggregated = encoder_meta
         else:
-            for i in range(len(meta)):
+            for i in range(len(aggregated)):
```

Comment:
It will not be faster to iterate directly on the value like

`` for I in aggregated`` ?